src/featureconverter.js

```diff
@@ -350,21 +350,37 @@ olcs.FeatureConverter.prototype.olLineStringGeometryToCesium = function(layer, f
     material: this.olStyleToCesium(feature, olStyle, true)
   });
 
-  // Handle both color and width
-  var outlineGeometry = new Cesium.PolylineGeometry({
+  var geometryOptions = {
     // always update Cesium externs before adding a property
     positions: positions,
     width: this.extractLineWidthFromOlStyle(olStyle),
     vertexFormat: appearance.vertexFormat
-  });
+  };
+
+  var outlinePrimitive;
+  var heightReference = this.getHeightReference(layer, feature, olGeometry);
+
+  if (heightReference == Cesium.HeightReference.CLAMP_TO_GROUND) {
```

Comment:
If the positions are 3D (just test the first one), could you disregard the `CLAMP_TO_GROUND` height reference?